caida libre 

Calcule la altura sobre la superficie h terrestre a la que debe estar un satélite que 
orbita la Tierra con período T. 
para esto debemos emplerar las ecuaciones de la ley gravitacion universal y la ley de 
movimiento circular las cuales son :
    
\begin{equation} \label{eq:ley gravitacion universal}
F_{g}=\frac{GmM}{r^{2}}
\end{equation}






\begin{equation} \label{eq:ley de movimiento para la fuerza centripeta}
F=\frac{m v^{2}}{r}
\end{equation}

ahora relacionamos las dos ecuaciones igualando las dos fuerzar y despejando el radio, obtemiendo la sigiente expresión :





\begin{equation} \label{eq:ecuacion para emplear ( tercera ley de keppler )}
r=(\frac{T^{2} G M}{4 \pi ^{2}})^{1/3}
\end{equation}


a partir de esto debemos tener en cuenta el radio de la tierra entonces para hallar la altura se le resta el radio:

\begin{equation} \label{eq:ecuacion para emplear ( tercera ley de keppler )}
h=(\frac{T^{2} G M}{4 \pi ^{2}})^{1/3} -r_{tierra}
\end{equation}




esta ecuación representa la altura del satelite en fuincion del periodo orbital 

entonces asigmanos los valores de cada una de las constantes, sabemos que el unico valor de entrada 
debe ser el periodo orbital del  satelite.

In [34]:
x=float(input("digite el periodo: "))


digite el periodo: 86400


digitamos segun el inciso (3) un periodo orbital de 5060 seg 

In [35]:
M=5.97e24
G=6.67e-11
pi=3.14
a=(x**2)*G*M
b=4*(pi**2)
r=6.37e6
R=(a/b)**(1/3)
total=R-r
total2=total/1000
t=round (total2,4)

In [36]:
print("la altura es de " ,t, " km")

la altura es de  35871.1877  km


para un periodo de 90 min (5400 seg), el satelite debe estar a un altura de 282.5702  km 
sobre la superficie de la tierra.

ahora para un periodo de 45 min (2700 seg), el satelite debe estar a un altura de -2179.1434  km,
lo que significa que el satelite no podria estar orbitando la tierra en este tiempo ya que una distancia
negativa indicaria que el satelite se encuentra en la superficie de la tierra.




 (c)
 para un periodo orbita de 86148 seg que equivale a un dia sideral la altura es de  35789.0121  km lo cual significa que el satelite dio una vuelta a la tierra.

para un periodo orbial de 86400 seg la altura edl satelñite sobre la superficie de la tierra es de 35871.1877  km, la diferencia de la altura es de 82.17 km que se obtiene restando las alturas de los periodos 24 horas y 23.93 horas.